pillar/flavors/production.sls

```diff
@@ -45,6 +45,9 @@
             "parallelism": 1,
             "taskmanager_mem_preallocate": false,
             "pyflink_yarn_container_count": 1
+        },
+        "graphite-api":{
+            "retentions": "60:10080"
```

Comment:
Production should have the highest retention of the three flavors.

Reply:
For production, have set retention 14 days.